kademlia/protocol.py

```diff
@@ -108,9 +116,9 @@ def handleCallResponse(self, result, node):
         """
         if result[0]:
             self.log.info("got response from %s, adding to router" % node)
-            self.router.addContact(node)
             if self.router.isNewNode(node):
```

Comment:
Yeah - good point - I think that can move too.  Maybe the new method could be something like ```welcomeIfNewNode``` or something like that just to indicate there's probably a few things going on.  Thanks for the help!